web-ui/src/pages/FeedbackRequestPage.jsx

```diff
@@ -135,14 +151,31 @@ const FeedbackRequestPage = () => {
     }
     query.step = `${activeStep + 1}`;
     history.push({...location, search: queryString.stringify(query)});
-  }, [canProceed, activeStep, steps.length, query, location, history, handleSubmit]);
+
+  }, [canProceed, activeStep, steps.length, query, location, history]);     // eslint-disable-line react-hooks/exhaustive-deps
 
   const onBackClick = useCallback(() => {
     if (activeStep === 1) return;
     query.step = `${activeStep - 1}`;
     history.push({...location, search: queryString.stringify(query)});
   }, [activeStep, query, location, history]);
 
+    const sendFeedbackRequest = async(feedbackRequest) => {
+          if (csrf) {
+                    let res = await createFeedbackRequest(feedbackRequest, csrf);
+                    let data =
+                      res.payload && res.payload.data && !res.error
```

Comment:
Check for status 200 instead of (or in addition to) `!res.error`

Reply:
Resolved